.github/workflows/ruby.yml

```
@@ -0,0 +1,78 @@
+name: Ruby
+
+on:
+  pull_request:
+    branches: [master,staging]
+  push:
+    branches: [master,staging]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DB_PORT_5432_TCP_ADDR_TEST: localhost
+      DB_ENV_POSTGRESQL_USER_TEST: postgres
+      DB_ENV_POSTGRESQL_PASS_TEST: postgres
+      DB_ENV_POSTGRESQL_DB_TEST: postgres
+      RAILS_ENV: test
+
+    services:
+      postgres:
+        image: postgres:17
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - name: Check out files
+        uses: actions/checkout@v3
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+        env:
+          BUNDLE_GEMS__CONTRIBSYS__COM: ${{ secrets.BUNDLE_GEMS__CONTRIBSYS__COM }}
+
+
+      - name: Set up DB
+        run: |
+          bundle exec rails db:setup          
```

That loads your db/seeds.rb; do you need those for tests? If not, do this instead:
```suggestion
          bundle exec rails db:create
          bundle exec rails db:schema:load
```

---

db:setup replaced